web/src/message_events.js

```diff
@@ -491,7 +491,7 @@ export function update_messages(events) {
                 new_stream_id: post_edit_stream_id,
                 new_topic: post_edit_topic,
             });
-            unread.clear_and_populate_unread_mention_topics();
+            unread.clear_and_populate_unread_mention();
```

Comment:
Does it make better sense for these new functions to use the plural? `clear_and_populate_unread_mentions()`, `add_message_to_unread_mentions()`?

Reply:
Yes it is better